README.md

```diff
@@ -3,21 +3,25 @@
 Our guide to how 18F Engineering selects candidates, from
 resume screen to offer.
 
+**Note to 18F staff:**
+
+As much as possible, this should be publicly-accessible: interviews shouldn't
+be about surprises or gotchas. For the few things that can't be public,
+put them in a private Google Doc and link it here.
+
+## Build / Deployment
+
+This project uses CircleCI, here the dashboard
```

Comment:
is there a word missing here?

Reply:
well, yes, there are a couple things missing.  Fixed.